project/Version.scala

```diff
@@ -7,12 +7,12 @@ object Version {
   val version = "0.1.5"
   val floodmodel = version + either("FLOODMODEL_VERSION_SUFFIX", "-SNAPSHOT")
 
-  val scala = "2.10.6"
-  val geotrellis = "0.10.0-561030e"
+  val scala = "2.11.1"
```

Comment:
[GeoTrellis 1.0](https://github.com/locationtech/geotrellis/blob/1.0/project/Version.scala#L19) is on `2.11.8`